README.md

```diff
@@ -28,3 +34,41 @@ Step 7: yarn run start:dev
 - [Node.js](https://nodejs.org/en/) - Tested against LTS
 - NPM (installed with Node.js) - Works with versions 2 and 3
 - [Redis server](http://redis.io/download) running on the default port
+
+Then visit: [http://localhost:8080/](http://localhost:8080/) For the start page and applicant journey
+
+
+## Skip email verify step
+
+You can skip the email authentication locally or in some of the testing environments.  You'll need to make sure you have an environment variable `allowSkip=true`. You'll also need an email as part of save and return.  You have 3 options either: using a `skipEmail` environment variable; using a key value paramenter in the url; or both.
+
+1. To use an email environment variable, you'll need to set it like so `skipEmail=sas-hof-test@digital.homeoffice.gov.uk`. You can then go to the following url.
```

Comment:
Not sure if we should be showing any env variable publicly, better to use a placehold like: 

`skipEmail=<email>`

you can then use <email> placeholder instead  in the url